### DISCUSSION TITLE
Account lost

### DISCUSSION THREAD

**Discussion by Shubham-8817** [community/community#88202] [Discussions] [answered]
### Select Topic Area

Question

### Body

Hello all,

I had my email ID associated with GitHub. After logging in, I did not verify the email. Now, when I try to log in, I realize I don't have the password. I am unable to reset my password because I haven't verified my account. I want to access my GitHub account.

**Comment by PrashantRatanparikhe** [comment, marked as answer]
Hi, 
Please, open the GitHub Support form at https://support.github.com/contact. then select "account" option, and send your request to github support.
